cfme/common/host_views.py

```diff
@@ -252,7 +250,21 @@ def entity_class(self):
 
 class HostsView(ComputeInfrastructureHostsView):
     toolbar = View.nested(HostsToolbar)
-    sidebar = View.nested(HostSideBar)
+
+    @View.nested
+    class filters(Accordion):  # noqa
+        ACCORDION_NAME = "Filters"
+
+        @View.nested
+        class Navigation(VerticalNavigation): # noqa
```

Comment:
**OPTIONAL**
I'm not sure why the class name starts with capital `N`? `view.filters.Navigation.select('ALL')` looks not so nice as `view.filters.navigation.select('ALL')`

Reply:
make sense